### ISSUE TITLE
@scope fluent packages 

### ISSUE THREAD

**Issue by romulocintra** [projectfluent/fluent.js#274]
I would suggest to scope the fluent namespace  : 

Example : 
```
@fluent-js/react
@fluent-js/dom
//OR 
@fluent/intl-polyfill
```
This would simplify npm search from all fluent related components

**And when you search fluent** 
https://www.npmjs.com/search?q=fluent

instead of unique package i would appear all related or scoped packages example : https://www.npmjs.com/search?q=%40ionic

**Comment by stasm** [comment]
Sadly, `@fluent` is taken. Some time I reached out to the person who owns it, but I haven't heard back. I created the `@projectfluent` org with the intent to start using it at some point in the future. I haven't yet figured out how disruptive such change would be nor how to make it smooth. Do you have experience with doing it? Would you recommend to rename all packages at once and publish them using the same version numbers? Or bump the versions?

**Comment by romulocintra** [comment]
Hi @stasm  , i would suggest  the following steps 👍 

**1 -**  Deprecation message for next releases(repository command line tools etc).  
        Example : Version 1.0.0 of fluent would be publish under the @projectfluent/ or @fluent-js/ scope

**2 -**  Release next versions under both scoped and not scoped till you planned deprecation 


Dont do step 2 for to long. 

After achieve the scoped of all packages archive old packages and add deprecation messages.

Would be my suggestion for this process 

PS : @projectfluent/  i don't see it as an easy name @project-fluent/ looks better for me :)

**Comment by stasm** [comment]
I finally got around to doing this. For each package on the following list, I first published a new minor update of the package under the old name. The, I published the same code under the new name, and I deprecated the old package on npm.

- `fluent` is now `@fluent/bundle` 0.13,
- `fluent-syntax` is now `@fluent/syntax` 0.14,
- `fluent-react` is now `@fluent/react` 0.9.0,
- `fluent-sequence` is now `@fluent/sequence` 0.4.0,
- `fluent-langneg` is now `@fluent/langneg` 0.3.0.

There are two packages remaining: `fluent-dom` and `fluent-gecko`.

- `fluent-gecko` doesn't need to be renamed, because it's not published to npm. I'd still make it explicit by adding `private: true` to `package.json`.
- `fluent-dom` has a few unpublished changes on master. @zbraniecki Could you please release a new version of it (minor or patch, both work) under the current name? i can then help with the renaming to `@fluent/dom` if you'd like, or feel free to do it yourself.

**Comment by stasm** [comment]
With @zbraniecki's help, I've just published `@fluent/dom` 0.5.0. All packages are now published under the `@fluent` scope.
